app/views/includes/opening-times.nunjucks

```diff
@@ -21,8 +41,8 @@
 {% endif %}
 {% endmacro %}
 
-{{ openingTimes(gp.openingTimes.reception, 'Reception opening times', 'No information available. Contact reception to find out opening times.','Open (Reception)') }}
-{{ openingTimes(gp.openingTimes.surgery, 'GP appointment times', "We don't have opening times for this surgery. Contact reception to find out when the surgery is open.",'Open (GP)') }}
+{{ openingTimes(gp.openingTimes.reception, 'Reception opening times', 'No information available. Contact reception to find out opening times.','Open (Reception)',gp.exceptionalTimes.alterations,'Changes to opening times','Changes (Reception)','callout callout--warning') }}
+{{ openingTimes(gp.openingTimes.surgery, 'GP appointment times', "We don't have opening times for this surgery. Contact reception to find out when the surgery is open.",'Open (GP)',nil,'','','') }}
```

Comment:
Alterations here are against the reception only, when in reality in the back end data they are against neither - you have an openingTimes object with child objects reception, surgery, alterations, i.e. alterations apply to both?

If the reception times were missing but surgery times and alterations were available, then the exceptional times would not show.

I think the exceptional times belong in it's own macro, which would cut down on the number or parameters to the existing macro, and all the empty values passed to the surgery times.